<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableViewer</class>
 <widget class="QDialog" name="TableViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accounting Assisant</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:repeat, x1:0.531, y1:1, x2:0.525584, y2:0, stop:0 rgba(0, 180, 255, 255), stop:0.482955 rgba(70, 92, 124, 255), stop:1 rgba(0, 0, 0, 246));</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>511</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="TableTitle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(0, 0, 0, 0));
color: rgb(240, 255, 255);
font-size: 36,</string>
   </property>
   <property name="text">
    <string>Table Window Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="savebutton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(255,255,255,200);    
color: rgb(0, 0, 0); 
font: 12pt;
font-family: &quot;Microsoft Sans Serif&quot;;
font-weight: 400;
border-radius: 7px;
}
QPushButton:hover{ 
background-color:rgba(255,255,255,255);
}</string>
   </property>
   <property name="text">
    <string>Retornar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
